packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -469,7 +469,6 @@ export default function TypographyPanel( {
 						hasFontWeights={ hasFontWeights }
 						fontFamilyFaces={ fontFamilyFaces }
 						size="__unstable-large"
-						__nextHasNoMarginBottom
```

Comment:
Removing `__nextHasNoMarginBottom` since it's not even a valid prop anymore for `CustomSelectControl`. The component doesn't have a bottom margin anyway